### PR TITLE
feat: Add GetFloatingClientMetadata method

### PR DESCRIPTION
**Pull request by muneebkq**
GetFloatingClientMetadata, to retrieve the value of the floating client metadata. The method takes a key as an argument and returns the corresponding value.

### CODE REVIEW

File: cryptlex/lexfloatclient/lexfloatclient_native.py [hunk 1]
```diff
@@ -159,6 +159,11 @@ def byte_to_string(input):
 GetFloatingClientMeterAttributeUses.argtypes = [CSTRTYPE, POINTER(c_uint32)]
 GetFloatingClientMeterAttributeUses.restype = c_int
 
+GetFloatingClientMetadata = library.GetFloatingClientMetadata
+GetFloatingClientMetadata.argtypes = [CSTRTYPE, STRTYPE, c_uint32]
+GetFloatingClientMetadata.restype = c_int
+
+
 RequestFloatingLicense = library.RequestFloatingLicense
```
Comment (ahmad-kemsan) [inline cryptlex/lexfloatclient/lexfloatclient_native.py:167]: @muneebkq Why this extra line?

File: cryptlex/lexfloatclient/lexfloatclient.py [hunk 2]
```diff
@@ -267,6 +267,28 @@ def GetFloatingClientMeterAttributeUses(name):
...
+
+        Args:
+                key (str): metadata key to retrieve the value
+
+        Raises:
+                LexFloatClientException
+
+        Returns:
+                str: value of metadata for the key
+        """
```
Comment (ahmad-kemsan) [inline cryptlex/lexfloatclient/lexfloatclient.py:282]: @muneebkq Update the docs, it does not return the value of metadata for the key, instead it simply returns `value of the floating client metadata`

**Review by ahmad-kemsan — CHANGES_REQUESTED** [review]
Please check the comments

File: cryptlex/lexfloatclient/lexfloatclient.py (hunk 2 above)
Comment (muneebkq) [reply cryptlex/lexfloatclient/lexfloatclient.py:282]: added

File: cryptlex/lexfloatclient/lexfloatclient_native.py (hunk 1 above)
Comment (muneebkq) [reply cryptlex/lexfloatclient/lexfloatclient_native.py:166]: removed

File: cryptlex/lexfloatclient/lexfloatclient.py [hunk 3]
```diff
@@ -267,6 +267,28 @@ def GetFloatingClientMeterAttributeUses(name):
...
+                key (str): metadata key to retrieve the value
+
+        Raises:
+                LexFloatClientException
+
+        Returns:
+                str: value of the floating client metadata
+        """
+        cstring_key = LexFloatClientNative.get_ctype_string(key)
+        buffer_size = 256
```
Comment (ahmad-kemsan) [inline cryptlex/lexfloatclient/lexfloatclient.py:284]: @muneebkq Updated the buffer size to 4096

**Review by ahmad-kemsan — APPROVED** [review]

**Pull request merged**